libs/galley-types/src/Galley/Types.hs

```diff
@@ -392,6 +392,7 @@ data Connect = Connect
 
 data MemberUpdateData = MemberUpdateData
     { misOtrMuted       :: !(Maybe Bool)
+    , misOtrMutedStatus :: !(Maybe MutedStatus)
```

Comment:
You probably would have caught that first-try had there been a comment discussing the difference between this and `MemberUpdate` on both types.  :-)

Reply:
;)